ldk/go/examples/search-global/loop/loop_test.go

```diff
@@ -0,0 +1,43 @@
+package loop_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	ldk "github.com/open-olive/loop-development-kit/ldk/go"
+	loop "github.com/open-olive/loop-development-kit/ldk/go/examples/search-global/loop"
+	ldktest "github.com/open-olive/loop-development-kit/ldk/go/ldk-test"
+)
+
+func TestController(t *testing.T) {
```

Comment:
I think this test needs to do more. I would like to see it simulate the user entering text in the global search. Then confirming that it gets a whisper in response. 

Take a look at the whisper-confirm example as a guide. I think a staging approach like the one in that example would be good here.
Ex:
Stage 1 - Simulate user searching text
Stage 2 - Confirm whisper is received with correct content